Abha_Rana/Tic-Tac-Toe/tt.css

```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  .container {
+    display: flex;
+    background: #fff;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
it is recommended to use shorthand for writing CSS declaration improve readability
for eg. use flex-flow instead as below

flex-flow: <flex-direction> <flex-wrap>;

Reply:
Issue fixed.  used flex-flow instead of flex-direction.